weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallsPage/CallsTableButtons.tsx

```diff
@@ -68,6 +77,25 @@ export const ExportSelector = ({
     callQueryParams.gridSort
   );
 
+  // TODO(gst): tabulate correct list of ref columns
```

Comment:
We currently determine whether or not a ref is expandable on a value-by-value basis. Ie, there is no way to tell if a given column contains refs, without checking every single value. As implemented, this is just a very inclusive heuristic that allows all columns that are equal to or start with one of the json columns. I believe these are the only columns that contain expandable refs. The current sever implementation won't error if you pass a column to expand that has a non-ref value (heterogeneous data means that _some_ might be refs, others not), so this at least shouldn't error.